### ISSUE TITLE
Let'sEncrypt Certificates may not be updating automatically as they should

### ISSUE THREAD

**Issue by jacobdgm** [DDMAL/CantusDB#1174]
I got an email a couple of days ago pointing out that our certificates for cantusdatabase.org are going to expire in a few weeks. 

I thought we had a cron job set up to automatically renew these. I can't remember where we put it, and a brief search of a few likely places hasn't turned it up. We need to figure out the part of our code that is supposed to handle this and review it to make sure it's working properly.

**Comment by jacobdgm** [comment]
... this may be a duplicate of https://github.com/DDMAL/CantusDB/issues/870 (i.e., we may not have actually set this up)

**Comment by lucasmarchd01** [comment]
@jacobdgm and I spent a bit of time investigating this issue. Checking the docker logs on production, we found that this message was logged every ~1min:
```
Saving debug log to /var/log/letsencrypt/letsencrypt.log
Certbot doesn't know how to automatically configure the web server on this system. However, it can still get a certificate for you. Please run "certbot certonly" to do so. You'll need to manually configure your web server to use the resulting certificate.
```
Observations:
- The Certbot container is restarting every few seconds.
- The path /var/log/letsencrypt/letsencrypt.log doesn't exist, indicating an issue with saving logs.

Possible next steps:
- Update the Certbot service in your Docker Compose file to include a specific Certbot command to obtain a certificate without attempting automatic web server configuration.
- Ensure that the volumes certbot_data and certificates have correct permissions, allowing the Certbot container to write data to them.
